package.json

```diff
@@ -28,7 +28,8 @@
     "hh:test": "hardhat test",
     "hedera:start": "npx @hashgraph/hedera-local start --limits=false --dev=true --balance=10000000",
     "hedera:stop": "npx @hashgraph/hedera-local stop",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "besu:start": "docker run -d -v ./utils/besu-configs:/var/lib/besu/ -p 8540:8545 -p 8541:8546 hyperledger/besu:24.6.0 --config-file=/var/lib/besu/customConfigFile.toml"
```

Comment:
Q: Is there a README that should be updated and explains how besu is being used here?

Reply:
I missed adding a README, it's my mistake. Maybe a better idea is to remove this from the npm scripts section and just move the `docker run` command into the `opcode-logger-testing.yml` only? In that way, we'll not confuse the developers with this specific besu testing flow. Or it's better to add a descriptive README for `besu:start` npm script?

Reply:
Add a README to be more descriptive for this flow.
Approach makes sense, just explain it somewhere so it's an easy reference